File: Latihan B/Python/Disk.py
class Disk():

	#atribut private
	iType = "" #tipe disk
	iCapacity = 0 #kapasitas disk
	iPrice = 0 #harga disk


	#konstruktor
	def __init__(self, iType = "", iCapacity = 0, iPrice = 0):
		self.iType = iType
		self.iCapacity = iCapacity
		self.iPrice = iPrice

	#getter dan setter atribut iType
	def setType(self, iType):
		self.iType = iType

	def getType(self):
		return self.iType

	#getter dan setter atribut iCapacity
	def setCapacity(self, iCapacity):
		self.iCapacity = iCapacity

	def getCapacity(self):
		return self.iCapacity

	#getter dan setter atribut iPrice
	def setPrice(self, iPrice):
		self.iPrice = iPrice

	def getPrice(self):
		return self.iPrice

File: Latihan A/Disk.cpp
class Disk{

	private:
		//atribut private
		string type; //tipe disk
		int capacity; //kapasitas disk
		int price; //harga disk

	public:

		//konstruktor
		Disk(){

		}

		Disk(string t, int c, int p){
			this->type = t;
			this->capacity = c;
			this->price = p;
		}

		//getter dan setter atribut type
		void setType(string t){
			this->type = t;
		}

		string getType(){
			return this->type;
		}

		//getter dan setter atribut capacity
		void setCapacity(int c){
			this->capacity = c;
		}

		int getCapacity(){
			return this->capacity;
		}

		//getter dan setter atribut price
		void setPrice(int p){
			this->price = p;
		}

		int getPrice(){
			return this->price;
		}

		//destruktor
		~Disk(){

		}

};